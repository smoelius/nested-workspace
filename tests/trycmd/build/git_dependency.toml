args = ["build", "-vv", "--offline"]

[bin]
name = "cargo"

[fs]
cwd = "../../../fixtures/git_dependency"
