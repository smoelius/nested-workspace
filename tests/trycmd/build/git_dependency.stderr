...
   Compiling git_dependency v0.1.0 ([CWD])
...
[git_dependency 0.1.0] <<< [CWD]/package
...
[git_dependency 0.1.0]    Compiling package v0.1.0 ([CWD]/package)
...
[git_dependency 0.1.0]     Finished `dev` profile [unoptimized + debuginfo] target(s) in [..]s
[git_dependency 0.1.0] >>> [CWD]/package
...
    Finished `dev` profile [unoptimized + debuginfo] target(s) in [..]s
